assets/js/modules/analytics/dashboard/dashboard-widget-all-traffic.js

```diff
@@ -69,6 +69,20 @@ class AnalyticsAllTraffic extends Component {
 
 		const dataError   = ( error || ! receivingData );
 		const wrapperClass = dataError ? 'googlesitekit-nodata' : '';
+		const {
```

Comment:
@lucymtc 
I recommend to move this checking part to a new method in this component class.
So the render() method is easier to read.

Reply:
thanks, done in commit https://github.com/google/site-kit-wp/pull/84/commits/185c25614a6446ec73bb1df04212ca87818e9c58